platform/ios/src/MGLMapView.mm

```diff
@@ -209,7 +209,7 @@ @interface MGLMapView () <UIGestureRecognizerDelegate,
 @property (nonatomic) UILongPressGestureRecognizer *quickZoom;
 @property (nonatomic) UIPanGestureRecognizer *twoFingerDrag;
 
-@property (nonatomic) MGLCameraChangeReason cameraChangeReason;
+@property (nonatomic) MGLCameraChangeReason cameraChangeReasonBitmask;
```

Comment:
Consider `currentCameraChangeReason` as the name for this property. “Bitmask” is unnecessary detail, because `MGLCameraChangeReason` is already a bitmask.